tests/e2e/Makefile

```diff
@@ -25,9 +25,25 @@ e2e-test:
 		echo "[ERR] KUBECONFIG is missing, must be set"; \
 		exit 1; \
 	fi
-	$(GO_CMD) test --tags=e2e -v . -args \
+	$(GO_CMD) test -failfast --tags=e2e -v . -args \
 		-kubeconfig=$(KUBECONFIG) \
```

Comment:
The failfast flag is required to ensure that the test suite fails after 1st failed test.